src/patternfly/components/ContextSelector/examples/context-selector-example.hbs

```diff
@@ -1,5 +1,5 @@
 {{#> context-selector id="context-selector-collapsed-example" context-selector--label-text="Selected Project"}}
-  {{#> context-selector-toggle}}
+  {{#> context-selector-toggle context-selector-toggle--attribute='id="context-selector-expanded-example-toggle" aria-labelledby="context-selector-expanded-example-toggle context-selector-expanded-example-label"'}}
```

Comment:
We should give the context selector an id: "context-selector--id" and then reuse that in the context-selector-toggle  as the ID's need to be unique.  For example you can look at the table.
it would be:

`{{#> context-selector-toggle context-selector-toggle--attribute=(concat 'id="' context-selector--id '-toggle" aria-labelledby="context-selector-expanded-example-toggle context-selector-expanded-example-label"')}}`